docs/algorithms/mopt/algorithm_en.md

```diff
@@ -0,0 +1,46 @@
+# MOpt
+
+## What is MOpt
+
+[MOpt-AFL](https://github.com/puppet-meteor/MOpt-AFL) is an example implementation for MOpt[^usenix19] paper.
+We think their algorithm is not even a customized PSO, but original heuristics inspired by PSO.
+This is because the original PSO and MOpt's PSO-like algorithms don't share the problem setting and the objective function at all.
+This can be emphasized to some extent considering that I saw some people misunderstand this point.
+They optimize the selection of mutation operators to find more interesting inputs by generating probability distribution from particles optimized with PSO-like algorithm.
```

Comment:
This part sounds a bit too aggressive and assertive to me for some reason. 
Moreover, if we say "we think", it seems like we are making some claims or clarifying our standpoint, which a third party cannot tell true or not. But, as a matter of fact, as I explained, this is true, so I would suggest the following:  

```suggestion
It tries to optimize the selection of mutation operators to find more interesting inputs by generating probability distributions from *particles* calculated with its original heuristics inspired by PSO (Particle Swarm Optimization).
```

Reply:
thanks for great suggestion. reflected in 3f78ce24c2055672e02e9fa58ca2c19b4b453345